### ISSUE TITLE
[Nodejs] session.sessionState.lastAccess is always the current timestamp

### ISSUE THREAD

**Issue by matheus208** [Microsoft/BotBuilder#916]
Using Node.js lib 3.2.1 

Hi,

I'm trying to use a middleware to expire the user dialog if they have been inactive for too long.

I'm trying this:

```
bot.use({
        botbuilder: function (session, next) {
                const shouldExpire = (s) => (Date.now() - s.sessionState.lastAccess) > TIME_IDLE_TO_RESET_SESSION);
                if(shouldExpire(session)) expire();
                ...
```

However, `Date.now() - s.sessionState.lastAccess` is always 0. I was expecting lastAccess to be the last time the user has messaged the bot?

**Comment by Stevenic** [comment]
Good catch... I'll move where that's getting set to be after all session middleware runs.

**Comment by Stevenic** [comment]
Just committed that change. Will go out in the next update. Likely Monday if not sooner. For now... I'd just add your own timestamp to the sessionState to track when you last saw the session.

**Comment by Stevenic** [comment]
Should be fixed in v3.2.3. Please let me know if it's not and I'll re-open this issue.
